environment.yml

```diff
@@ -4,59 +4,36 @@ channels:
   - defaults
 
 dependencies:
-  - ale >=0.8.0
-  - boost=1.68.0
-  - armadillo
-  - blas
-  - bullet=2.86.1=0
-  - bz2file
-  - bzip2
-  - cmake >=3.15
-  - cspice
-  - curl
-  - doxygen
-  - eigen
-  - embree
-  - geos>=3.7,<3.8
-  - geotiff
-  - gmm
-  - gmp
+  # Huge Depedencies 
+  - qt
+  - gdal>=3
```

Comment:
Gdal and ISIS share a lot of dependencies, it's basically easier to just include gdal rather than maintaining a bigger list. This only makes sense if we plan to incorporate Gdal into ISIS itself (which I believe is part of the plan moving forward), otherwise, I can remove this and expand it out with more specific dependencies.

Reply:
I understand allowing gdal to pull in xerces, but the last time we did this that didn't allow xalan (requries older version of xerces) to be pulled in and I don't see xalan in the list anymore. xalan is required to build the documentation. We don't see this because we are lazy and don't build the docs often.

Reply:
Thanks for pointing it out, updated with Xalan-c.

Reply:
For the environmeny.yml we can do this, but for the meta.yml we don't need xalan if we include gdal.